### PR TITLE
basic game

### PR DESCRIPTION
**Pull request by Javier-Pedrozo**
Here is a basic version of the game. You can win with X or O, or the game can end in a draw.

https://github.com/user-attachments/assets/551f0cd3-a0ac-4272-9e1c-74947467b8e0

https://github.com/user-attachments/assets/dc6c815c-40fe-4709-8080-643f86aebf94

https://github.com/user-attachments/assets/3d91071d-6ca2-442a-a0fc-807e33a26867

### CODE REVIEW

File: react-tutorial/public/index.html [hunk 1]
```diff
@@ -1,43 +1,11 @@
...
-</html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <div id="root"></div>
+</body>
+</html>
```
Comment (jimeemk) [inline react-tutorial/public/index.html]: New line here (files should end with an empty line)

File: react-tutorial/src/index.js [hunk 2]
```diff
@@ -1,17 +1,12 @@
...
     <App />
-  </React.StrictMode>
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+  </StrictMode>
+);
```
Comment (jimeemk) [inline react-tutorial/src/index.js]: New line

File: react-tutorial/src/App.js [hunk 3]
```diff
@@ -1,25 +1,81 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
 
-function App() {
```
Comment (jimeemk) [inline react-tutorial/src/App.js:4]: The "App" component is quite standard on a React App.  I recommend you to create a new "components" folder and to move the Square and the Board to different files of that folder. Then, in App, you can use the Board component and everything should work.

**Review by jimeemk — CHANGES_REQUESTED** [review]
Great job 😎 ! I left some comments!

File: react-tutorial/public/index.html [hunk 4]
```diff
@@ -1,43 +1,11 @@
...
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (jimeemk) [inline react-tutorial/public/index.html]: The title is the name of the tab in the browser. You could change it to "tic tac toe"

File: react-tutorial/src/Board.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+      <div className="board-row">
+        <Square value={squares[6]} onSquareClick={() => handleClick(6)} />
+        <Square value={squares[7]} onSquareClick={() => handleClick(7)} />
+        <Square value={squares[8]} onSquareClick={() => handleClick(8)} />
+      </div>
+    </>
+  );
+}
+
+function calculateWinner(squares) {
```
Comment (jimeemk) [inline react-tutorial/src/Board.js]: I would move this function to line 21, inside Board

File: react-tutorial/src/Board.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
```
Comment (jimeemk) [inline react-tutorial/src/Board.js]: In order to have a better structure I recommend you to move Board and Square to a "components" folder

File: react-tutorial/public/index.html (hunk 4 above)
Comment (Javier-Pedrozo) [reply react-tutorial/public/index.html]: Done.

File: react-tutorial/src/Board.js (hunk 5 above)
Comment (Javier-Pedrozo) [reply react-tutorial/src/Board.js]: Done.

File: react-tutorial/src/styles.css [hunk 7]
```diff
@@ -0,0 +1,90 @@
+* {
```
Comment (jimeemk) [inline react-tutorial/src/styles.css]: These styles should go in the component folder. For example the classes related to Board should be in /Board in a file called "styles.css". Then you should import that file from the component.js

File: react-tutorial/src/components/square/index.js [hunk 8]
```diff
@@ -0,0 +1 @@
+export { default } from "./Square";
```
Comment (jimeemk) [inline react-tutorial/src/components/square/index.js]: The component folder usually starts with uppercase

File: react-tutorial/src/components/board/index.js [hunk 9]
```diff
@@ -0,0 +1 @@
+export { default } from "./Board";
```
Comment (jimeemk) [inline react-tutorial/src/components/board/index.js]: You don't need two files (one for the export and one for the component), please delete this one and rename Board.js to index.js

**Review by jimeemk — APPROVED** [review]
Great job!! 💯

**Pull request merged**